src/client/pages/FormBuilder.tsx

```diff
@@ -19,13 +19,7 @@ const WithNavBar: FC = ({ children }) => (
   <Flex direction="column" minH="100vh" bgColor="neutral.200">
     <Navbar />
     <HelpButton />
-    <Container
-      h="calc(100vh - 80px)"
-      overflow="auto"
-      mt="80px"
-      maxW="100vw"
-      pb="32px"
-    >
+    <Container h="calc(100vh - 73px)" overflow="auto" mt="73px" maxW="100vw">
```

Comment:
```suggestion
    <Container
      h="calc(100vh - 73px)"
      overflow="auto"
      mt="73px"
      maxW="100vw
      pb="32px"
    >
```
Hey Kee Wei, I think you might have accidentally undone the bottom padding fix here